### DISCUSSION TITLE
Augmentation in Yolov5

### DISCUSSION THREAD

**Discussion by TasneemSanuri19** [ultralytics/ultralytics#2267] [Q&A]
does anyone know how i can disable augmentation for yolov5 classification?

**Comment by glenn-jocher** [comment]
@TasneemSanuri19 hello! It looks like you're asking about YOLOv5, but this is the YOLOv8 repository. However, to answer your question in the context of YOLOv8, if you want to disable augmentations during training for classification or any other task, you can adjust the augmentation settings in the YAML configuration file for your dataset or training run. Set the augmentation parameters to minimal values or disable specific augmentations as needed. For more detailed guidance on how to configure your training, please refer to the documentation at https://docs.ultralytics.com. Happy training! 😊🚀
